README.md

```diff
@@ -21,14 +21,19 @@ NOTE: CARAT was developed for crystallographic groups in dimensions up to 6.
 Most algorithms also work in higher dimensions. However, integer overflow
 is not trapped in general.
 
+## Website
+
+You can find the official CARAT homepage and documentation on:
+[https://lbfm-rwth.github.io/carat/](https://lbfm-rwth.github.io/carat/)
 
 ## Installation
 
 The easiest way to compile CARAT is to go to the CARAT base directory
-(the one containing this REAMDME), and issue the following
+(the one containing this README.md), and issue the following commands:
+     ./autogen.sh (only if you are building CARAT directly from GitHub)
```

Comment:
I'd not do it this way. First off, it can't be copy&pasted, so a comment should be preceded by `#` inside a shell script quote.

But secondly, I'd not mix the general instructions with the instructions for rare power users who use CARAT directly from the repository. Because many people will miss this comment which comes *after* the command to enter. Just like it is a bad idea to write "PRESS THE RED BUTTON but only in case of an emergency"; better to say "In case of an emergency, press the red button".

Calling `autogen.sh` in a release of CARAT can actually cause errors (if the user doesn't have autoconf/automake installed), which will confuse users, and would not have happend if the user had just done `./configure && make`. Worse, if only e.g. `autoconf` is installed but not `automake`, then running this command might end in a broken state that the user can only recover from by deleting and re-extracting the CARAT interface.

Hence, I'd separate the mention of `./autogen.sh` from the other build command. Just like I did in the `README.md` for GAP.

BTW, there is also a point for telling users to enter `./configure && make` instead of telling them two commands: This way, if `./configure` fails, then `make` is not run. Sometimes people don't realize there was an error in `configure`, then blindly proceed to enter `make`, and then report the error from *that* to us, which isn't very helpful, and thus requires an extra round of communication to find out what *actually* went wrong.
Note that the GAP `README.md` doesn't do this (yet). One reason for that is that we also instruct people about using various options for `configure`, and for that, it's easier to have the `make` separate. Also, I don't claim that the instructions in the GAP `README.md` are perfect; they are certainly not. As always, it's difficult to find the right balance between explaining enough and too much (and in general, getting this perfect is impossible anyway, as it depends on the reader)

Reply:
Thanks @fingolfin , you're right. I wasn't happy with the way I wrote things up, but I didn't really have a better way. I tried to change it now, I'm again open to suggestions.

Reply:
@fingolfin Are you happier with this now? Still open to suggestions. :)